esmvaltool/cmorizers/data/cmor_config/GPCP-SG.yml

```
@@ -0,0 +1,18 @@
+---
+# Common global attributes for Cmorizer output
+attributes:
+  dataset_id: GPCP-SG
+  version: '2.3'
+  tier: 2
+  modeling_realm: atmos
+  project_id: OBS
+  source: 'https://psl.noaa.gov/data/gridded/data.gpcp.html'
```

```suggestion
  source: 'https://downloads.psl.noaa.gov/Datasets/gpcp/precip.mon.mean.nc
```

I think the actual file location fits better here.